kotlin-codegen/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    kotlin("jvm")
+    kotlin("kapt")
+    kotlin("plugin.serialization")
+    id("com.squareup.sqldelight")
+    id("com.apollographql.apollo")
```

Comment:
We need to separate kapt from fast compiler plugins codegen so testing of non kapt stuff is still fast.

One acceptable thing is just to have a `kapt-codegen` module, and have the rest in the main project, or in a `kotlin-codegen` module, where we write that we have no kapt here.

Reply:
Done, only kapt is now in the codegen module

I verified that the others don't have an impact on the incremental build
https://gradle.com/s/2yhdxtxpqouiw

Reply:
Perfect!